net-analyzer/gvmd/files/gvmd-21.4.5-startpre.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Greenbone Vulnerability Manager Systemd ExecStartPre
```

Comment:
This script is also used by the OpenRC init, hence the comment that it's the "Systemd ExecStartPre" is slightly misleading. Maybe

```suggestion
# Greenbone Vulnerability Manager Systemd ExecStartPre / OpenRC start_pre()
```